src/components/Navbar/navbar.css

```diff
@@ -21,6 +21,10 @@
     justify-content: flex-end;
     cursor: pointer;
 }
+.navLink{
+    text-decoration: none;
+    color: #fff;
```

Comment:
Can use css variables.

Reply:
I will do it after completing the whole ui